src/main/webapp/app/overview/course-exams/course-exam-attempt-review-detail/course-exam-attempt-review-detail.component.html

```diff
@@ -0,0 +1,47 @@
+<div *ngIf="studentExam" (click)="openStudentExam()">
+    <!-- To clearly indicate a StudentExam within the working time, which can be resumed, the card should be displayed in blue -->
+    <div
+        [class]="
+            'row card-body justify-content-center card-general-settings ' +
+            (withinWorkingTime ? 'bg-primary text-white' : '') +
+            (withinWorkingTime || studentExam.submitted ? ' clickable' : '')
+        "
```

Comment:
Use the object notation of ``ngClass`` here
```suggestion
        [ngClass]="{ 'row card-body justify-content-center card-general-settings': true, 
            'bg-primary text-white': withinWorkingTime,
            'clickable': withinWorkingTime || studentExam.submitted }"
```